src/main/java/uk/ac/ebi/eva/pipeline/parameters/AnnotationParameters.java

```diff
@@ -63,6 +63,9 @@
     @Value(PARAMETER + JobParametersNames.APP_VEP_NUMFORKS + END)
     private String vepNumForks;
 
+    @Value(PARAMETER + JobParametersNames.APP_VEP_TIMEOUT + END)
+    private Long timeout;
```

Comment:
I launched it several times in the terminal, I think it's not a problem anymore (chunk size is an Integer as well). I'll change the others to number types